### DISCUSSION TITLE
Compositions in CSS-Modules

### DISCUSSION THREAD

**Discussion by codcodea** [vercel/next.js#33009] [Help]
### What version of Next.js are you using?

v12.0.7

### What version of Node.js are you using?

v16.13.0

### What browser are you using?

Opera 82.0.4227.33 

### What operating system are you using?

macOs

### How are you deploying your application?

next start

### Describe the Bug

CSS-Modules do not support compositions according to the referred standard https://github.com/css-modules/css-modules .

.className {
  color: green;
  background: red;
}

.otherClassName {
  composes: className;
  color: yellow;
}

Would be nice!

### Expected Behavior

Support of CSS-Modules

### To Reproduce

Add a MyComponent.module.css with composition

.className {
  color: green;
  background: red;
}

.otherClassName {
  composes: className;
  color: yellow;
}

**Comment by balazsorban44** [comment]
Following your reproduction steps, it works as expected.

**Reply by codcodea** [reply]
Okey thanks sorry to file inaccurate report. In my implementations of next.js app "composes" is not recognized.
